src/main/msp/msp.c

```diff
@@ -809,6 +809,13 @@ static bool mspCommonProcessOutCommand(uint8_t cmdMSP, sbuf_t *dst, mspPostProce
         sbufWriteU8(dst, 0);
 #endif // USE_OSD_STICK_OVERLAY
 
+        // API >= 1.42
+#ifdef USE_RX_LINK_QUALITY_INFO
```

Comment:
As per convention in Betaflight, new features should be added CLI only for the first release, so this should only be added after 4.1 has been released.